assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -96,7 +102,7 @@ export const TakeActionBoxoutEditor = ({
     imageAlt: alt_text,
   });
 
-  const actPageOptions = actPageList.map(actPage => ({ label: actPage.title.raw, value: actPage.id }));
+  const actPageOptions = actPageList.map(actPage => ({ label: actPage.title?.raw || actPage.id, value: actPage.id }));
```

Comment:
This change is not needed to support campaign pages, however it should be fixed as it causes a confusing crash if any page/campaign page does not have a title.